.semaphore/semaphore.yml

```
@@ -0,0 +1,94 @@
+version: v1.0
+name: Traefik
+agent:
+  machine:
+    type: e1-standard-4
+    os_image: ubuntu1804
```

```suggestion
    os_image: ubuntu2004
```

Since the rest use 2004?

---

2004 only works with e1-standard-2 machines: https://docs.semaphoreci.com/ci-cd-environment/ubuntu-20.04-image/

Disregard.

---

it's related to the integration test and groups v2